component/class/defaults.yml

```diff
@@ -6,7 +6,7 @@ parameters:
         version: 17.7.1
       mariadb:
         source: https://charts.bitnami.com/bitnami
-        version: 10.1.3
+        version: 11.6.2
```

Comment:
Why do we update the chart version in an e2e PR?

Reply:
because e2e tests found an issue in old charts :D